client-apis/encrypt.md

```diff
@@ -113,68 +161,168 @@ This document refers to the output of the key derivation algorithm as the derive
 Note that if the key derivation algorithm is the [identity KDF](../framework/algorithm-suites.md#identity-kdf),
 then the derived data key is the same as the plaintext data key.
 
-### Message
+The frame length used in the procedures described below is the input [frame length](#frame-size),
+if supplied.
+If a frame length is not specified on input, this operation MUST choose a reasonable value to
+default to.
```

Comment:
I think that all implementations use 4K.
I can get down with this not being in the spec,
but I would like to be cognitive about that ;)